### PR TITLE
4.4.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-core?_sp.spa=pr-4.4.0-release](https://developer.blackbaud.com/skyux-core?_sp.spa=pr-4.4.0-release)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.4.0 (2021-03-04)
+
+- Added the `getWidth`, `syncMaxHeight` and `resetHeight` methods to the `SkyCoreAdapterService`. [#213](https://github.com/blackbaud/skyux-core/pull/213)
```
Comment (blackbaud-paulah) [inline CHANGELOG.md:3]: #213 Mentions adding methods **syncHeight**, resetHeight, and getWidth. Should syncHeight also be added to this description?

**Review by blackbaud-johnly — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md:3]: Nice eye! The method was originally called `syncHeight` but we changed it in that PR to be `syncMaxHeight`.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-core?_sp.spa=pr-4.4.0-release](https://developer.blackbaud.com/skyux-core?_sp.spa=pr-4.4.0-release)

**Pull request merged**